docs/crash_reporting.md

```diff
@@ -72,6 +72,104 @@ yarn add -D @datadog/datadog-ci
 npm install --save-dev @datadog/datadog-ci
 ```
 
+#### Automatically on each release build (React Native >= 0.69)
+
+Manually uploading your sourcemaps on every release build takes time and is prone to errors. We recommend automatically sending your sourcemaps every time you run a release build.
+
+Create a new script file named `datadog-sourcemaps.sh` at the root of your project, containing this:
+
+```shell
+#!/bin/sh
```

Comment:
should we add `set -e` here the same way it appears in the script for pre-0.69 versions?

Reply:
```suggestion
#!/bin/sh
set -e
```